### PR TITLE
Azure to alibaba vpn

### PR DESCRIPTION
**Pull request by jrudley**
In a comment on the pull request, include the following information and check off the list.

 - [ ] Mini bio
  - Soft capped to 75 words
 - [ ] Desired date of publishing
  - Make sure this matches the `date` in the front matter
 - [ ] Social media contact info
 - [ ] Redaction: (yes|no)yes
  - Allow redaction, editing or structure changes by technical writer staff
 - [ ] Comments: (yes|no)yes
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by jrudley** [comment]
@catlook Looks like the strider test failed. Not sure if I need to let you know

**Comment by catlook** [comment]
reviewing!

**Comment by catlook** [comment]
James,  can you make sure this image is in the right place and named right?  Build error:

Liquid Exception: Couldn't find file '2018-11-15-Azure-to-Alibaba-VPN/azconnection.png' in _posts/2018-11-15-Azure-to-Alibaba-VPN.md

File: _posts/2018-11-15-Azure-to-Alibaba-VPN.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+date: 2018-11-15 00:00
+comments: true
+author: Jimmy Rudley
+published: true
+authorIsRacker: true
+categories:
+    - Azure
+---
+
+Businesses today are becoming more multi-cloud than ever. When it comes to Sitecore deployments, being able to have fast page response times in China are becoming more critical. My goal is to create a secure site to site vpn tunnel between Azure and Alibaba. Once the tunnel is setup, I can then test out remote publishing target deployments.
```
Comment (catlook) [inline _posts/2018-11-15-Azure-to-Alibaba-VPN.md]: ... China is becoming ...
... secure site-to-site vpn ...

File: _posts/2018-11-15-Azure-to-Alibaba-VPN.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+![config1]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/abip1.png %})
+Enable the Advanced Configuration option to list the Ike configurations. Generate a 16bit pre-shared key and type it into the **Pre-Shared Key** text box. Select **Ikev2** for **Version**, **aes256** for **Encryption Algorithm**, **28800** for **SA Life Cycle (Seconds)**.Under the **IPsec Connections** label, select **aes256** and **28800** for **SA Life Cycle (seconds)** 
+![config2]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/abip2.png %})
+![config3]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/abip3.png %})
+
+In the Azure Portal, navigate to your **Local Network gateway** connection object. Select **Connections** from the blade then select **Add**. Give your connection a **name**, select your **VPN Gateway** and type in your pre-shared key that was used in Alibaba IKE Configuration.
+![localConnection]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/azconnection.png %})
```
Comment (catlook) [inline _posts/2018-11-15-Azure-to-Alibaba-VPN.md]: [looks like this image may be missing?]

File: _posts/2018-11-15-Azure-to-Alibaba-VPN.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+authorIsRacker: true
+categories:
+    - Azure
+---
+
+Businesses today are becoming more multi-cloud than ever. When it comes to Sitecore deployments, being able to have fast page response times in China are becoming more critical. My goal is to create a secure site to site vpn tunnel between Azure and Alibaba. Once the tunnel is setup, I can then test out remote publishing target deployments.
+
+<!-- more -->
+
+Starting off on the Azure side, we need to create a vnet with a non-overlapping address range to what our other cloud provider network will be using.
```
Comment (catlook) [inline _posts/2018-11-15-Azure-to-Alibaba-VPN.md]: ... side, you need to ...  [You switch back and forth between second-person singular voice ({you} create a new gateway) and first-person plural (we need to create a vnet).  When I notice this, I'm switching to second-person for consistency).

File: _posts/2018-11-15-Azure-to-Alibaba-VPN.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+![VNET]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/createvnet.png %})
+
+Once the vnet has been created, create a new **Gateway Subnet**. Now that a Gateway Subnet has been defined, create a **Virtual Network Gateway** in Azure. 
+Make sure to select **route-based** as the **VPN type**
+![GatewaySubnet]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/gatewaysubnet.png %})
+
+While the Azure VPN Gateway is provisioning, log into the Alibaba Cloud dashboard. At this point, we want to mimic the steps from Azure, but within Alibaba Cloud. Create a **VPC** and a **Vswitch** and do not overlap the IP address range used in Azure.
```
Comment (catlook) [inline _posts/2018-11-15-Azure-to-Alibaba-VPN.md]: ...  At this point, you want to mimic the steps from Azure but within Alibaba Cloud ...
... **Vswitch**, and do ...

File: _posts/2018-11-15-Azure-to-Alibaba-VPN.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+![VNET]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/createvnet.png %})
+
+Once the vnet has been created, create a new **Gateway Subnet**. Now that a Gateway Subnet has been defined, create a **Virtual Network Gateway** in Azure. 
+Make sure to select **route-based** as the **VPN type**
+![GatewaySubnet]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/gatewaysubnet.png %})
+
+While the Azure VPN Gateway is provisioning, log into the Alibaba Cloud dashboard. At this point, we want to mimic the steps from Azure, but within Alibaba Cloud. Create a **VPC** and a **Vswitch** and do not overlap the IP address range used in Azure. 
+![VPC]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/ab-vpc-vswitch.png %})
+
+Next, create a **VPN Gateway** and select the VPC you previously created.
```
Comment (catlook) [inline _posts/2018-11-15-Azure-to-Alibaba-VPN.md]: ... VPC that you ...

File: _posts/2018-11-15-Azure-to-Alibaba-VPN.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+![VPNGateway]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/abvpngw.png %})
+
+Within the Azure portal, create a Local Network Gateway. Take the public ip address from the Alibaba VPN Gateway and type it into the **IP address** text box in Azure. Specify the Alibaba VPC CIDR address space in the **Address space** text box. Select create to provision the resource
```
Comment (catlook) [inline _posts/2018-11-15-Azure-to-Alibaba-VPN.md]: ... public IP address ...
... text box. Click **Create** to provision the resource.

File: _posts/2018-11-15-Azure-to-Alibaba-VPN.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+![LocalGateway]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/alngw.png %})
+
+Within the Alibaba portal, create a **Customer Gateway**. This is similar to a Local Network Gateway in Azure. Type in the public ip address from Azure's VPN Gateway into the **IP Address** text box
```
Comment (catlook) [inline _posts/2018-11-15-Azure-to-Alibaba-VPN.md]: ... a *Local Network Gateway* in ...
... Type the public IP address ...
... text box.

File: _posts/2018-11-15-Azure-to-Alibaba-VPN.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+![CustomerGateway]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/abcustomergateway.png %})
+
+The next step is to configure the connections to both gateways. In Alibaba, select **Create an IPSec Connection** and choose the VPC and VPN Gateway. Type in the IP CIDR of your Alibaba local network in **Local Network** then type in the IP CIDR of your Azure Network in **Remote Network**. Slect **Yes** for Effective Immediately.
```
Comment (catlook) [inline _posts/2018-11-15-Azure-to-Alibaba-VPN.md]: ... **Local Network** and then type ...
...  Select **Yes** for the **Effective Immediately** choice.

File: _posts/2018-11-15-Azure-to-Alibaba-VPN.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+![LocalGateway]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/alngw.png %})
+
+Within the Alibaba portal, create a **Customer Gateway**. This is similar to a Local Network Gateway in Azure. Type in the public ip address from Azure's VPN Gateway into the **IP Address** text box
+![CustomerGateway]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/abcustomergateway.png %})
+
+The next step is to configure the connections to both gateways. In Alibaba, select **Create an IPSec Connection** and choose the VPC and VPN Gateway. Type in the IP CIDR of your Alibaba local network in **Local Network** then type in the IP CIDR of your Azure Network in **Remote Network**. Slect **Yes** for Effective Immediately. 
+![config1]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/abip1.png %})
+Enable the Advanced Configuration option to list the Ike configurations. Generate a 16bit pre-shared key and type it into the **Pre-Shared Key** text box. Select **Ikev2** for **Version**, **aes256** for **Encryption Algorithm**, **28800** for **SA Life Cycle (Seconds)**.Under the **IPsec Connections** label, select **aes256** and **28800** for **SA Life Cycle (seconds)**
```
Comment (catlook) [inline _posts/2018-11-15-Azure-to-Alibaba-VPN.md]: ... the **Advanced Configuration** option ...
... 16 bit, pre-shared ...
... **Encryption Algorithm**, and **28800** ...
... (Seconds)**. Under ...

File: _posts/2018-11-15-Azure-to-Alibaba-VPN.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+![config1]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/abip1.png %})
+Enable the Advanced Configuration option to list the Ike configurations. Generate a 16bit pre-shared key and type it into the **Pre-Shared Key** text box. Select **Ikev2** for **Version**, **aes256** for **Encryption Algorithm**, **28800** for **SA Life Cycle (Seconds)**.Under the **IPsec Connections** label, select **aes256** and **28800** for **SA Life Cycle (seconds)** 
+![config2]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/abip2.png %})
+![config3]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/abip3.png %})
+
+In the Azure Portal, navigate to your **Local Network gateway** connection object. Select **Connections** from the blade then select **Add**. Give your connection a **name**, select your **VPN Gateway** and type in your pre-shared key that was used in Alibaba IKE Configuration.
```
Comment (catlook) [inline _posts/2018-11-15-Azure-to-Alibaba-VPN.md]: ... blade and then select ...
... Gateway**, and type ...
... that you used  ...

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-0886fbde9a/)

**Comment by jrudley** [comment]
> James, can you make sure this image is in the right place and named right? Build error:
> 
> Liquid Exception: Couldn't find file '2018-11-15-Azure-to-Alibaba-VPN/azconnection.png' in _posts/2018-11-15-Azure-to-Alibaba-VPN.md

that was it! thanks

File: _posts/2018-11-15-Azure-to-Alibaba-VPN.md [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+![localConnection]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/azconnection.png %})
+
+The final step is to add a route entry into your Alibaba VPC Route Table. Within your VPC, edit the route table and add a new entry. Type in the remote network cidr block, set the **Next Hop Type** to **VPN Gateway** and select your VPN Gateway.
```
Comment (catlook) [inline _posts/2018-11-15-Azure-to-Alibaba-VPN.md]: ... network CIDR block ...
... Gateway**, and select ...

File: _posts/2018-11-15-Azure-to-Alibaba-VPN.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+![route]({% asset_path 2018-11-15-Azure-to-Alibaba-VPN/abroute.png %})
+
+At this point, we have a route based Site-to-Site vpn configured between Azure and Alibaba Cloud. I stood up a SQL Server and frontend Sitecore content delivery node and configured a remote publishing target. Benchmarking from US to China for a remote publish was not favorable due to random timeouts. Looking at the Sitecore publishing service, I deployed an Azure webapp and configured publishing that route. Publishing was instantly faster and did not have anymore timeouts.
```
Comment (catlook) [inline _posts/2018-11-15-Azure-to-Alibaba-VPN.md]: ... point, you have ...
... route-based, site-to-site VPN configured ...
... and a front-end Sitecore content-delivery node. Then I configured a remote publishing target. ...
... from the US to ...
... publishing for that route. ...
... faster and had no timeouts.

**Comment by catlook** [comment]
James, do you plan to make any of these suggested edits?

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-8dae8d4f2c/)

**Comment by jrudley** [comment]
All done, thanks again @catlook !

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-36527457fb/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-4b8e049e7b/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-20613c7cd2/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-080311f855/)

**Pull request merged**